assignments/lambda-classes.js

```diff
@@ -1 +1,206 @@
 // CODE here for your Lambda Classes
+
+class Person {
+  constructor(attr) {
+    this.name = attr.name,
+    this.age = attr.age,
+    this.location = attr.location,
+    this.gender = attr.gender
+  }
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}`;
+  }
+}
+
+class Instructor extends Person {
+  constructor(attr) {
+    super(attr);
+    this.specialty = attr.specialty,
+    this.favLanguage = attr.favLanguage,
+    this.catchPhrase = attr.catchPhrase
+  }
+  demo(subject) {
+    return `Today we are learning about ${subject}.`
+  }
+  grade(student, subject) {
+    return `${student.name} receives a perfect score on ${subject}.`
+  }
+  gradeRoulette(student) {
+    console.log(student.grade);
```

Comment:
This extra `console.log` logs the student's grade 1 extra time _before_ the grade is computed to a new value. Perhaps you forgot to remove this from testing?